packages/filebrowser/src/model.ts

```diff
@@ -786,6 +787,27 @@ export class FilterFileBrowserModel extends TogglableHiddenFileBrowserModel {
     this._filterDirectories = value;
   }
 
+  /**
+   * Whether to apply fuzzy filter.
+   */
+  get useFuzzyFilter(): boolean {
+    return this._useFuzzyFilter;
+  }
+  set useFuzzyFilter(value: boolean) {
+    this._useFuzzyFilter = value;
+    this._filterFileBrowserSettingsChanged.emit({ useFuzzyFilter: value });
+  }
+
+  /**
+   * Signal for settings changed
+   */
+  get filterFileBrowserSettingsChanged(): ISignal<
```

Comment:
I would suggest using a shorter name, we already know this pertains to file browser:

```suggestion
  get filterSettingsChanged(): ISignal<
```